docs/operate-a-stake-pool/hardware-requirements.md

```diff
@@ -9,11 +9,11 @@ The latest technical specifications and supported platforms can be found on the
 
 Currently the following specifications are recommended:
 
-- Servers: 1 for block producer node, at least 1 for relay node
+- Servers: 1 for block producer node, at least 1 as relay node
 - CPU: An Intel or AMD x86 processor with two or more cores, at 1.6GHz or faster (2GHz or faster for a stake pool or relay)
-- Memory: 16 GB of RAM
-- Storage: 100GB of free storage (150GB recommended for future growth)
-- Operating system: 64-bit Linux (i.e. Ubuntu Server 20.04 LTS)
+- Memory: 24GB of RAM
```

Comment:
Should we convert it to network-wise table (node doco [adopted](https://github.com/input-output-hk/cardano-node/blob/master/doc/getting-started/install.md#prerequisites), tho is outdated)

Reply:
I prefer tables in general, but there's no advantage to putting a table here because
1. The specs are the only thing on the page here, while the documentation page uses it as an introduction to a big load of text: so it works because the specs stand out as a "read this before you continue" to the setup instructions.
2. Particularly the Air Gap node needs to be highlighted for the Mainnet, while irrelevant for the testnets, so this item wouldn't be suited by the table format at all.
3. We have items here that aren't included on the table: these would widen it, make it less responsive, causing horizontal scrollbars and/or cramping in the text-heavy columns.
4. We can't just copy the Markdown for the table here, as we have in other doc sections, because we can see they allow the material to go out of date.

So since the testnet is only different in 2 scales, would you be happy just adding this?  I've updated it in the PR but please @rdlrt @os11k check the values since I have no experience on the testnets:

---

For testnet pools, some requirements are smaller:
- Memory: 4GB of RAM
- Storage: 20GB of free storage
- Air-gapped environment not required

---

This assumes 2 relays are still required on the testnets (I don't think this is strictly true?) and that Windows and Mac nodes are not permitted on testnets (can that really be true?).  I'm OK leaving this to the reader's discretion, but if we're ever going to split hairs it might as well be here. 😜

Reply:
IMO - 20GB is safe, as current usage on preview/preprod is ~5.5/3.4GB (+account for.near future and OS).
Happy with above